tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -113,6 +113,17 @@ static cl::list<std::string>
            cl::value_desc("+SPV_extenstion1_name,-SPV_extension2_name"),
            cl::ValueRequired);
 
+static cl::list<std::string> SPIRVAllowUnknownIntrinsics(
+    "spirv-allow-unknown-intrinsics", cl::CommaSeparated,
+    cl::desc(
+        "Unknown intrinsics that begin with any prefix from the "
+        "comma-separated "
+        "input list will be translated as external function calls in SPIR-V.\n"
+        "Leaving any prefix unspecified (default) would naturally allow "
+        "all unknown intrinsics"),
```

Comment:
I guess it can be formatted better, right?

Reply:
Fair point; I hope it is now!